src/Evolution/Executables/GeneralizedHarmonic/GeneralizedHarmonicBase.hpp

```diff
@@ -289,7 +289,8 @@ struct ObserverTags {
                                      non_tensor_compute_tags>;
 };
 
-template <size_t volume_dim, bool LocalTimeStepping>
+template <size_t volume_dim, bool LocalTimeStepping,
+          bool UseNumericalInitialData>
```

Comment:
Note: this template parameter will be deleted again in the next PR that actually factory-creates the initial data in the GH executables.